godot-core/src/builtin/basis.rs

```diff
@@ -156,6 +157,7 @@ impl Basis {
     /// Creates a [`Quaternion`] representing the same rotation as this basis.
     ///
     /// _Godot equivalent: `Basis()`_
+    #[doc(alias = "Basis")]
     pub fn to_quat(self) -> Quaternion {
```

Comment:
What does this equivalence mean? `Basis` is a constructor; but we're converting _to_ a quaternion. Doesn't seem to make sense to me.

Rather remove the "Godot equivalent" than have it wrong/confusing.

Reply:
actually i believe the correct doc-alias here would be `get_rotation_quaternion`